.htaccess

```diff
@@ -8,6 +8,39 @@ RewriteEngine On
 
 RedirectMatch permanent "^/((?!index|404|automation.*|ansible_community|collections|community|core.*|galaxy|lint|ecosystem|users|developers|maintainers|platform|ansible-prior*)[^/]+)\.html" "/ansible/$1.html"
 
+#####################################################################
+# This section replaces rules defined for older versions that
+# redirect plugin and module pages to collections. This section
+# also consolidates specific redirects that were defined in multiple
+# .htaccess configuration files.
+#####################################################################
+
+# Redirect plugin and module pages for devel and latest
+RedirectMatch permanent "^/ansible/(devel|latest)/(plugins|modules)/(.+)\.html$" "https://docs.ansible.com/collections.html"
+
+# Redirect plugin and module pages for versions 2.3 to 2.6 and 2.10
+RedirectMatch permanent "^/ansible/(2\.(10|[3-6]))/(plugins|modules)/(.+)\.html$" "https://docs.ansible.com/collections.html"
```

Comment:
```suggestion
RedirectMatch permanent "^/ansible/(2\.[3-6])/(plugins|modules)/(.+)\.html$" "https://docs.ansible.com/collections.html"
```

Reply:
We can remove the `2.10` redirects for `plugins` and `modules` subdirectories following on from https://github.com/ansible/docsite/pull/265